### PR TITLE
Blog app - controllers specs

### PR DESCRIPTION
**Pull request by Tekhlay**
### Mielstones 💪 

- [x] Write tests for controllers.

### CODE REVIEW

File: spec/requests/posts_spec.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe 'GET /show' do
+    it 'returns http success' do
+      get '/posts/show'
+      expect(response).to have_http_status(:success)
+    end
+  end
```
Comment (mckent05) [inline spec/requests/posts_spec.rb:16]: - According to the project requirements, the actions in your controller should test:
1. If response status was correct, which you have already done 👏 
2. If a correct template was rendered, not yet done
3. If the response body includes correct placeholder text, not yet done.

- Please fix all these issues 🙏

File: spec/requests/users_spec.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'GET /index' do
+    it 'returns http success' do
+      get '/users/index'
+      expect(response).to have_http_status(:success)
+    end
+  end
+end
```
Comment (mckent05) [inline spec/requests/users_spec.rb:10]: - According to the project requirements, the actions in your controller should test:
1. If response status was correct, which you have already done 👏 
2. If a correct template was rendered, not yet done
3. If the response body includes correct placeholder text, not yet done.

- Please fix all these issues 🙏

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @Tekhlay, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 
- You have done well adding test for your controllers 👏 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by prolajumokeoni — APPROVED** [review]
Hi @Tekhlay 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
 - All changes requested have been implemented ✅ 
 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**